packages/eas-cli/src/graphql/queries/UpdateQuery.ts

```diff
@@ -54,7 +57,7 @@ export const UpdateQuery = {
         .toPromise()
     );
   },
-  async viewBranchAsync({ appId, name }: Pick<ViewBranchUpdatesQueryVariables, 'appId' | 'name'>) {
+  async viewBranchAsync({ appId, name, limit }: ViewBranchUpdatesQueryVariablesWithOptionalLimit) {
```

Comment:
alternatively, instead of making limit an optional argument, we could just create a second gql query that does not include updates at all:
```
query ViewBranchUpdates($appId: String!, $name: String!) {
              app {
                byId(appId: $appId) {
                  id
                  updateBranchByName(name: $name) {
                    id
                    name
                    }
                  }
                }
              }
```

I chose to set the limit to 0 to make the smallest change on the API. Seemed to get all the benefits of a new query without the boilerplate of making/maintaining a new query.